ui/src/dataLoaders/components/lineProtocolWizard/configure/__snapshots__/LineProtocolTabs.test.tsx.snap

```diff
@@ -12,11 +12,16 @@ exports[`LineProtocolTabs rendering renders! 1`] = `
       ]
     }
   />
-  <Grid>
-    <GridRow>
-      <GridColumn
+  <t
```

Comment:
I wonder if we should disable minification in clockface (renaming variables like `Grid` to some random identifier like `t`). It would make these snapshot tests a little more robust

Reply:
I agree it's a little too cryptic for human consumption

Reply:
https://github.com/influxdata/clockface/issues/77